src/get-started/timestamping-tutorial.md

```diff
@@ -70,35 +79,32 @@ where we indicate the external libraries required for our service.
 ??? note "Imports"
 
     ```rust
-    extern crate chrono;
-    #[macro_use]
     extern crate exonum;
     extern crate exonum_time;
     #[macro_use]
+    extern crate exonum_derive;
+    #[macro_use]
     extern crate failure;
     #[macro_use]
     extern crate log;
-    extern crate serde;
     #[macro_use]
     extern crate serde_derive;
```

Comment:
If we won't change rust edition in `Cargo.toml` to 2015 then `extern crate ...` is redundant.